### PR TITLE
pubsub: convert to minitest

### PR DESCRIPTION
**Pull request by quartzmo**
closes: #673

### CODE REVIEW

File: pubsub/app.yaml [hunk 1]
```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-runtime: ruby
-env: flex
+runtime: ruby27
 entrypoint: bundle exec ruby listener.rb
 liveness_check:
```
Comment (dazuma) [inline pubsub/app.yaml:17]: I think we can ditch this liveness_check for app engine standard.

**Review by dazuma — CHANGES_REQUESTED** [review]

File: pubsub/acceptance/subscriptions_test.rb [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper"
+require_relative "../subscriptions.rb"
+
+describe "subscriptions" do
+  let(:pubsub) { Google::Cloud::Pubsub.new }
+  let(:topic_name) { random_topic_name }
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/acceptance/subscriptions_test.rb:20]: Great catch!

File: pubsub/README.md [hunk 3]
```diff
@@ -33,24 +33,31 @@ environment variable to the path to the key file, for example:
...
 ### Set Project ID
 
-Next, set the `GOOGLE_CLOUD_PROJECT` environment variable to the project name
-set in the [Google Cloud Platfrom Developer COnsole](https://console.cloud.google.com):
+Next, set the *GOOGLE_CLOUD_PROJECT* environment variable to the project name
+set in the
+[Google Cloud Platform Developer Console](https://console.cloud.google.com):
 
-    export GOOGLE_CLOUD_PROJECT="YOUR-PROJECT-ID"
+    `export GOOGLE_CLOUD_PROJECT="YOUR-PROJECT-ID"`
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/README.md:40]: Agreed. I copied this upper part of the README without modification from google-cloud-ruby. So this should also be changed in:

- google-cloud-language/samples/README.md
- google-cloud-storage/samples/README.md
- google-cloud-text_to_speech/samples/README.md
- google-cloud-vision/samples/README.md

File: pubsub/README.md [hunk 4]
```diff
@@ -33,24 +33,31 @@ environment variable to the path to the key file, for example:
 
 ### Set Project ID
 
-Next, set the `GOOGLE_CLOUD_PROJECT` environment variable to the project name
-set in the [Google Cloud Platfrom Developer COnsole](https://console.cloud.google.com):
+Next, set the *GOOGLE_CLOUD_PROJECT* environment variable to the project name
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/README.md:36]: Agreed. I copied this upper part of the README without modification from google-cloud-ruby. So this should also be changed in:

- google-cloud-language/samples/README.md
- google-cloud-storage/samples/README.md
- google-cloud-text_to_speech/samples/README.md
- google-cloud-translate/samples/README.md
- google-cloud-vision/samples/README.md

File: pubsub/Gemfile [hunk 5]
```diff
@@ -18,8 +18,11 @@ gem "google-cloud-pubsub"
 gem "sinatra"
 
 group :test do
+  gem "google-style", "~> 1.24.0"
```
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/Gemfile:21]: Good call, got ahead of myself here.

File: pubsub/acceptance/subscriptions_test.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/acceptance/subscriptions_test.rb]: Fixed.

File: pubsub/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/README.md]: Fixed in this repo.

File: pubsub/Gemfile (hunk 5 above)
[the remark this answers is not in the document]
Comment (quartzmo) [reply pubsub/Gemfile]: Done.

File: pubsub/app.yaml (hunk 1 above)
Comment (quartzmo) [reply pubsub/app.yaml]: Removed, tested manually. works.

**Review by dazuma — APPROVED** [review]

**Pull request merged**